omnisafe/utils/plotter.py

```diff
@@ -61,14 +62,14 @@ def __init__(self) -> None:
 
     def plot_data(
         self,
-        sub_figures,
-        data,
-        xaxis='Steps',
-        value='Rewards',
-        condition='Condition1',
-        smooth=1,
-        **kwargs,
-    ):
+        sub_figures: np.ndarray,
+        data: list,
```

Comment:
list[what]?

Reply:
Thanks for your hints. I will fill all those blank dict/tuple/list in future commits.